### PR TITLE
Update markdown docs to release on first Tuesday of every month

### PR DESCRIPTION
**Pull request by valentijnscholten**
in master for immediate effect

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ See [Wrappers](WRAPPERS.md)
 
 ## Release and branch model
 
-We release every month on the last Tuesday. These release are minor releases, i.e. 2.0.0 to 2.1.0. In between there might be bugfix/hotfix/patch releases when needed, i.e. 2.0.1.
+We release every month on the frist Tuesday. These release are minor releases, i.e. 2.0.0 to 2.1.0. In between there might be bugfix/hotfix/patch releases when needed, i.e. 2.0.1.
```
Comment (StefanFl) [inline README.md:72]: There is a typo if **first**

**Review by StefanFl — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (StefanFl) [reply README.md:72]: And another one in the next sentence. It should be _These release**s**_

Comment (valentijnscholten) [reply README.md]: fixed

**Comment by valentijnscholten** [comment]
Feel free to fix typo's or change the wording if you feel to.

**Review by StefanFl — APPROVED** [review]

**Comment by devGregA** [comment]
I think this should go to dev because I think this will create merge conflicts. I need to do some additional readme cleanup so I can take valentijn's text and add it into my next PR. Trying to align docs with the readme and lean a little more on the docs site.

**Comment by valentijnscholten** [comment]
@devGregA I didn't see you your overhaul PR of the README. In this case it's better to do this PR against dev indeed. Will update.

**Comment by valentijnscholten** [comment]
closing as there are too many conflicts, will recreate against dev.

**Pull request closed without merging**